tools/cassandra_delete_range/main.go

```diff
@@ -480,11 +546,14 @@ func performDeleteQueries(cluster *gocql.ClusterConfig, info *deleteInfo, colSet
 
 	close(chunksChannel)
 
-	wg.Add(len(chunks))
-	sessionCreationWaitGroup.Add(len(chunks))
+	wg.Add(numberOfParallelClientThreads)
+	sessionCreationWaitGroup.Add(numberOfParallelClientThreads)
+
+	query := info.Query
+	bindCount := strings.Count(query, "?")
 
-	for i := 0; i < len(chunks); i++ {
-		go func() {
+	for i := 0; i < numberOfParallelClientThreads; i++ {
```

Comment:
I'm not sure whether these are threads, I think the loop is launching go routines.  It's very small thing, but perhaps renaming the variable to `workerCount` or something similar might avoid confusion.

Reply:
Yea, could do. It's what it was in the original example from Scylla (link at the top).